test/run-tests.py

```diff
@@ -534,7 +547,13 @@ def RunTest(info, options, variables, verbose_level=0):
   cwd = REPO_ROOT_DIR
   gen_input_path = info.CreateInputFile()
   rel_gen_input_path = os.path.relpath(gen_input_path, cwd)
-  variables['out_dir'] = os.path.dirname(rel_gen_input_path)
+
+  out_dir = os.path.splitext(rel_gen_input_path)[0] + "_out"
```

Comment:
I just wanted to be sure that nothing was still using /tmp.   With this change the test will never delete their intermediate files.